### PR TITLE
fix: fallback to clipboard when text cannot be extracted from running application

### PR DESCRIPTION
**Pull request by iloveitaly**
## Description
If the text fails to be extracted from the running application (happens alot with vs code for me) then fallback to the clipboard.

## Screencast
NA

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by mil3na** [comment]
Hi! Thank you so much for your contribution to Raycast!
My name is Milena and I will be reviewing your update today :)

Selection and clipboard are two different things. May I suggest adding another command that targets the clipboard instead?

**Comment by iloveitaly** [comment]
@mil3na 

1. The extension used to be this way and it looks like it was recently refactored to merge these operations together
2. This PR does not change the existing functionality (the same command picks from selected text + clipboard)—it simply fixes the currently-broken functionality of the command

tl;dr I agree with you, but this is a bugfix on existing functionality and the extension author seems to have wanted the commands merged for some reason.

**Review by mil3na — APPROVED** [review]
Yes, you are correct. Sorry for my misunderstanding. LGTM.
